src/Ardalis.Extensions/Math/Minus.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace Ardalis.Extensions.Math
+{
+    // Naming: https://en.wikipedia.org/wiki/Template:Arithmetic_operations
+    public static partial class MathExtensions
+    {
+        public static int Minus(this short minuend, short subtrahend)
```

Comment:
Same as for add, if we have `int Minus(int, int)` then we should have `short Minus(short, short)` I think.

Reply:
Same as Add. 
See: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/arithmetic-operators
And: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/expressions#numeric-promotions